modules/pointcloudio/include/opencv2/load_point_cloud.hpp

```diff
@@ -0,0 +1,76 @@
+#ifndef __OPENCV_LOAD_POINT_CLOUD_HPP__
+#define __OPENCV_LOAD_POINT_CLOUD_HPP__
+
+#include <opencv2/core.hpp>
+#include <iostream>
+
+//////////////////////////////// pont cloud codec ////////////////////////////////
+namespace cv
+{
+namespace pc
```

Comment:
No need to create new namespace for public API calls. Internals should be there, it's ok.